BraveRewards/Updating the BraveRewards framework.md

```diff
@@ -0,0 +1,85 @@
+## Prerequisites
```

Comment:
Could you link to this file somewhere in repo's readme?
Maybe below `Building the code`